pkg/minikube/cruntime/cruntime_test.go

```diff
@@ -141,6 +146,81 @@ func (f *FakeRunner) Run(cmd string) error {
 
 // docker is a fake implementation of docker
 func (f *FakeRunner) docker(args []string, root bool) (string, error) {
+	switch cmd := args[0]; cmd {
+	case "ps":
+		// ps -a --filter="name=apiserver" --format="{{.ID}}"
+		if args[1] == "-a" && strings.HasPrefix(args[2], "--filter") {
+			filter := strings.Split(args[2], `"`)[1]
+			fname := strings.Split(filter, "=")[1]
+			ids := []string{}
+			f.t.Logf("Looking for containers matching %q", fname)
```

Comment:
I'm wondering if prefixing these log messages with "fakeDocker" or similar would help in debugging when tests fail. These look too real :)

Reply:
Done.